lib/orocos/process.rb

```diff
@@ -783,6 +804,16 @@ def spawn(options = Hash.new)
                 module_bin = "#{pkg.exec_prefix}/bin/#{name}"
             end
             cmdline = [module_bin]
+
+            # check arguments for log_level
+            log_level = nil
+            if options[:log_level]
+                if [:debug, :info, :warn, :error, :fatal, :disable].include? options[:log_level]
+                    log_level = options[:log_level].to_s.upcase
+                else
+                    raise ArgumentError, "'#{options[:log_level]}' is not a valid log level."
```

Comment:
Display the list of valid log levels in the error message, that makes for a much more useful message.